bangazonapi/views/paymenttype.py

```diff
@@ -81,10 +81,10 @@ def list(self, request):
         """Handle GET requests to payment type resource"""
         payment_types = Payment.objects.all()
 
-        customer_id = self.request.query_params.get('customer', None)
+        customer = request.auth.user
```

Comment:
You'll probably want to get the customer object by the user that's logged in instead. https://github.com/nss-day-cohort-48/bangazon-NoahHusain/blob/b38473f231d85716695ca3f3442033a9c9905339/bangazonapi/views/paymenttype.py#L39

Reply:
Changes have been made to reflect this change!